velox/docs/functions/spark/datetime.rst

```diff
@@ -23,6 +23,10 @@ These functions support TIMESTAMP and DATE input types.
         SELECT add_months('2015-01-30', -2); -- '2014-11-30'
         SELECT add_months('2015-03-31', -1); -- '2015-02-28'
 
+.. spark:function:: current_date() -> date
+
+    Returns the current date at the start of query evaluation.
```

Comment:
Does the returned date start from 1970-01-01, could we add some description on that?

Reply:
This returns the current date so not sure if 1970-01-01 is relevant here?